### DISCUSSION TITLE
Conflict with Nginx Proxy Manager Preventing Coolify App Access

### DISCUSSION THREAD

**Discussion by gltched-usr** [coollabsio/coolify#2728] [Q&A]
I am unable to use the Traefik proxy buildin, because I already have Nginx Proxy Manager running on the same server, which occupies ports 80 and 443. My domain is pointed to my server, and I have set up the reverse proxy (cloud.<domain> and *.cloud.<domain>) to forward traffic to the Coolify container. However, the apps are not accessible under the domains assigned by Coolify; instead, the dashboard is displayed. How can I resolve this issue?

**Comment by sisou** [comment]
This here mentions you can run NPM in a container inside Coolify: https://github.com/coollabsio/coolify/discussions/2558

**Reply by SpuriousGer** [reply]
I'm in the same boat and I think essentially everyone using Proxmox is. We should have a solution for this.

**Reply by judeibe** [reply]
> I'm in the same boat and I think essentially everyone using Proxmox is. We should have a solution for this.

Using Proxmox SDN I created private network for coolify and then in a different container connected to my router I have NPM proxying traffic to coolify. It has worked well.

**Reply by gltched-usr** [reply]
@judeibe I dont have any experience regarding Proxmox, can you maybe help me?

**Reply by YazanKittaneh** [reply]
You guys get it working? @SpuriousGer

**Reply by djsisson** [reply]
It sounds like you are forwarding traffic to coolify which is just the ui

You still need to run the proxy but pick different ports 81/444

Forward traffic to this proxy and it will handle all the routing. You will have to use either dns challenge or install your certs as you won't be able to generate certs using http challenge

**Reply by Recreabox** [reply]
I have a similar case.

I installed my web hosting panel for WP management on my server.
Next, i installed Coolify.

But i can't use the builtin reverse proxy on Coolify because the ports 80 and 443 are used by my web hosting panel.
